indico/web/templates/forms/date_widget.html

```diff
@@ -19,6 +19,8 @@
             allowClear: {{ (field.allow_clear if field.allow_clear else false) | tojson }},
             earliest: {{ (field.earliest_date.isoformat() if field.earliest_date else none) | tojson }},
             latest: {{ (field.latest_date.isoformat() if field.latest_date else none) | tojson }},
+            weekendDisabled: {{ field.weekend_disabled | default(false) | tojson }},
+            disabledDates: {{ (field.disabled_dates if field.disabled_dates else none) | tojson }},
```

Comment:
```suggestion
            disabledDates: {{ field.disabled_dates | tojson }},
```

That should be sufficient, no?